java/java.lsp.server/test/unit/src/org/netbeans/modules/java/lsp/server/protocol/ServerTest.java

```diff
@@ -151,6 +151,7 @@ public ServerTest(String name) {
 
     @Override
     protected void setUp() throws Exception {
+        System.setProperty("java.awt.headless", Boolean.TRUE.toString());
```

Comment:
I was hoping that, if we integrated #2685, this change would no longer be needed. But that is probably not true, as the branding would only be active in integration tests of `nbcode` suite.

One would have to turn the branding on manually in the test...